.readthedocs.yml

```diff
@@ -1,4 +1,20 @@
-requirements_file: requirements.txt
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+version: 2
 
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: true
+
+# Set the version of Python and requirements required to build your docs
 python:
-   version: 3.5
+  version: 3.7
+  system_packages: true
```

Comment:
not sure whether this can cause any issues, I suppose it won't.

Reply:
I am actually not sure what gets installed as "system packages" but I have that option for all my projects and it doesn't seem to hurt. :woman_shrugging:

Reply:
OK, that's fine then. I guess  it's also the default, as in some weird cases we run into some version issues with their system installed packages (as I recall all could be traced back to some real issues with the packages/dependencies).